mitodl/mit-open
# open resources in new tab

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4713

### Description (What does it do?)
Resource drawer call to action ("Take Course", "Listen to podcast") opens in new tab now.

### How can this be tested?
1. Open the resource drawer (Click any resource... horizontal or vertical card... e.g., on homepage).
    - actually, don't click a video, they don't have the call to action links.
2. Click "Take Course" / "Take Program" or "Listen to podcast". It should open in a new tab.
